File: problem_thirty_nine.py
# The sum of the streams
inp = int(input("Enter a number:"))
inp2 = int(input("Enter power :"))
def MyFunction(inp,inp2):
    count = 0
    for i in range(0,inp2+1,1):
        value = inp**i
        count += value
    return count
print(MyFunction(inp,inp2)   )  

File: problem_thirty_nine.js
// The sum of the streams
var inp1 = 2
var inp2 = 5
function Myfunction(inp1,inp2){
    var count = 0
    for(var i =0 ;i< inp2 + 1; i++){
        var value = inp1**i
        count += value
    }
    return count

}
console.log(Myfunction(inp1,inp2))